### PR TITLE
[BobStuff] Motherbase shipment cost lowering, fixing 3 dumb bugs

### PR DESCRIPTION
**Pull request by CinderWC**
## About The Pull Request
Lowered the cost of the motherbase shipment from 20tc to 13tc, because the price was way too salty for what was essentially worth 4 + 1 + 1 + 1 + 8 (my personal CQC worth). Added soft padding to the kit to sweeten the deal even more. Added custom arm sprite by Junglerat for the CQC arm.

Fixes: https://github.com/Skyrat-SS13/Skyrat13/issues/1560
Shield bash will now only work in combat mode
Fixes: https://github.com/Skyrat-SS13/Skyrat13/issues/1589
Paint remover will actually remove paint. Black paint description objectively bettered.
Fixes: https://github.com/Skyrat-SS13/Skyrat13/issues/1591
polychromic shorts will now display your snowflake joints

## Why It's Good For The Game
nobody will buy a wack 20tc bundle, this makes it actually a fine deal. also bugfixing is good.

## Changelog
:cl:
balance: Motherbase shipment now costs 13tc and has soft padding included.
fix: Paint remover now removes paint.
fix: Shield bash now actually requires combat mode.
fix: Polychromic shorts will now display digi legs
imageadd: CQC arm icons.
/:cl:

### CODE REVIEW

File: code/game/objects/items/shields.dm [hunk 1]
```diff
@@ -33,10 +33,14 @@
 	return TRUE
 
 /obj/item/shield/alt_pre_attack(atom/A, mob/living/user, params)
+	if(!(user.combat_flags & COMBAT_FLAG_COMBAT_ACTIVE)) //Combat mode has to be enabled for shield bashing
+		return
 	user_shieldbash(user, A, user.a_intent == INTENT_HARM)
 	return TRUE
 
-/obj/item/shield/altafterattack(atom/target, mob/user, proximity_flag, click_parameters)
+/obj/item/shield/altafterattack(atom/target, mob/living/user, proximity_flag, click_parameters)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/shields.dm:41]:
```suggestion
/obj/item/shield/altafterattack(atom/target, mob/living/user, proximity_flag, click_parameters) //Skyrat change
```

File: code/modules/clothing/under/shorts.dm [hunk 2]
```diff
@@ -4,6 +4,7 @@
 	gender = PLURAL
 	body_parts_covered = GROIN
 	fitted = NO_FEMALE_UNIFORM
+	mutantrace_variation = STYLE_DIGITIGRADE|STYLE_NO_ANTHRO_ICON
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/under/shorts.dm:7]:
```suggestion
	mutantrace_variation = STYLE_DIGITIGRADE|STYLE_NO_ANTHRO_ICON //Skyrat change
```

File: code/modules/clothing/under/shorts.dm [hunk 3]
```diff
@@ -35,7 +36,6 @@
 	desc = "95% Polychrome, 5% Spandex!"
 	icon_state = "polyshortpants"
 	item_state = "rainbow"
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/under/shorts.dm:38]:
```suggestion
	item_state = "rainbow"
	//mutantrace_variation = NONE //Skyrat change
```

File: code/modules/clothing/under/shorts.dm [hunk 4]
```diff
@@ -47,7 +47,5 @@
 	desc = "Topless striped panties. Now with 120% more polychrome!"
 	icon_state = "polypantsu"
 	item_state = "rainbow"
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/under/shorts.dm:49]:
```suggestion
	item_state = "rainbow"
	//body_parts_covered = GROIN //Skyrat change
	//mutantrace_variation = STYLE_DIGITIGRADE|STYLE_NO_ANTHRO_ICON //Skyrat change
```

**Comment by CinderWC** [comment]
Support for this PR has been discontinued. All the work and bugfixing has been concluded and you may wish to merge or close this as your own leisure.

**Comment by CinderWC** [comment]
Bob wants to mirror this PR under his handle and (possibly) work on it more.

**Pull request closed without merging**